src/portalbackend/PortalBackend.DBAccess/Repositories/OfferSubscriptionsRepository.cs

```diff
@@ -81,7 +81,8 @@ sorting switch
                     ServiceName = g.Name,
                     CompanySubscriptionStatuses = g.OfferSubscriptions
                         .Where(os => os.OfferSubscriptionStatusId == statusId)
-                        .Select(s => new CompanySubscriptionStatusData(s.CompanyId, s.Company!.Name, s.Id, s.OfferSubscriptionStatusId, s.Company!.Address!.CountryAlpha2Code, s.Company!.BusinessPartnerNumber, s.Requester!.Email))
+                        .Select(s => new CompanySubscriptionStatusData(s.CompanyId, s.Company!.Name, s.Id, s.OfferSubscriptionStatusId, s.Company!.Address!.CountryAlpha2Code, s.Company!.BusinessPartnerNumber, s.Requester!.Email)),
+                    Image = g.Documents.Where(document => document.DocumentTypeId == DocumentTypeId.APP_LEADIMAGE && document.DocumentStatusId != DocumentStatusId.INACTIVE).Select(document => document.Id).FirstOrDefault()
```

Comment:
The check should be image status == locked, otherwise you'd be able to receive an pending image

```suggestion
                    Image = g.Documents.Where(document => document.DocumentTypeId == DocumentTypeId.APP_LEADIMAGE && document.DocumentStatusId == DocumentStatusId.LOCKED).Select(document => document.Id).FirstOrDefault()
```

Reply:
fixed

Reply:
apparently my remark about default-value of Guid being Guid.Empty is not fixed.

Reply:
fixed